taxi/models.py

```diff
@@ -0,0 +1,23 @@
+from django.conf import settings
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True, null=True)
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(
+        Manufacturer, on_delete=models.CASCADE, related_name="cars"
+    )
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name="cars")

```

Comment:
Good job on using `settings.AUTH_USER_MODEL` instead of `User` in your `ManyToManyField`. This is a best practice and it's nice to see it being followed here.